supertokens_python/recipe/dashboard/recipe.py

```diff
@@ -133,6 +133,7 @@ async def handle_api_request(
         path: NormalisedURLPath,
         method: str,
         response: BaseResponse,
+        user_context: Dict[str, Any],
```

Comment:
you need to pass in the user_context to the functions in this API. See node PR

Reply:
Fixed.